tests/fastcs/panda/db/panda.db

```diff
@@ -441,7 +441,7 @@ record(stringin, "$(IOC_NAME=PANDAQSRV):SEQ1:_PVI")
     field(VAL,  "$(IOC_NAME=PANDAQSRV):SEQ1:PVI")
     info(Q:group, {
         "$(IOC_NAME=PANDAQSRV):PVI": {
-            "value.seq1.d": {
+            "value.seq[1].d": {
```

Comment:
@callumforrester this diff is the change to the PVI structure as published over PVA.

Once inside the IOC, instead of publishing:
```yaml
value:
  seq1:
    d: PANDA:SEQ1:PVI
  seq2:
    d: PANDA:SEQ2:PVI
```
we now publish
```yaml
value:
  seq:
    - null
    - d: PANDA:SEQ1:PVI
    - d: PANDA:SEQ2:PVI
```

Horrific eh?

Reply:
Yuck, but also more structure so yay

Reply:
https://github.com/PandABlocks/PandABlocks-ioc/pull/139